### PR TITLE
jenkins: exec static-tests on slave instead of master

### PR DESCRIPTION
**Pull request by cgundogan**
Execute the static-tests on a slave instead of a master. See [1].

[1] https://github.com/RIOT-OS/RIOT/pull/6390#issuecomment-273187329

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -7,21 +7,28 @@ def driver_tests = []
 def pkg_tests = []
 def periph_tests = []
 def other_tests = []
-def unittests = []
+def unittests = [];
```
Comment (smlng) [inline Jenkinsfile]: why have a `;` here, but above not?

Comment (cgundogan) [inline Jenkinsfile]: ah, thanks for the hint. Still doing some tests here. Will ping you when I'm ready (:

File: Jenkinsfile [hunk 2]
```diff
@@ -53,9 +51,18 @@ node ('master') {
...
     }
+}
+
+stage('static-tests') {
+    node('linux && boards') {
+        deleteDir()
+
+        sh """git init; git remote add origin https://github.com/RIOT-OS/RIOT;
+        git fetch -u -n origin master:master pull/${env.CHANGE_ID}/merge:pull_${env.CHANGE_ID}
+        git checkout pull_${env.CHANGE_ID}"""
```
Comment (smlng) [inline Jenkinsfile]: maybe put this in a function, e.g. `fetchMaster`, too?

Comment (cgundogan) [inline Jenkinsfile]: but it's not just fetching master, it's also fetching the PR itself

Comment (smlng) [inline Jenkinsfile]: I see, name it differently, not `fetchMaster` 😄

**Comment by cgundogan** [comment]
@smlng I unified the fetch command

**Pull request merged**